acme/acme/client.py

```diff
@@ -631,7 +635,7 @@ def _add_nonce(self, response):
                 decoded_nonce = jws.Header._fields['nonce'].decode(nonce)
             except jose.DeserializationError as error:
                 raise errors.BadNonce(nonce, error)
-            logger.debug('Storing nonce: %r', decoded_nonce)
+            logger.debug('Storing nonce: %s', nonce)
```

Comment:
The nonce isn't really an "object" to be decoded. It's just a string of random bytes, base64 encoded. So the decoded values are not useful.